content/command-line/concepts/bash/terms/input-output-redirection/input-output-redirection.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Bash: Input-Output Redirection'
```

Comment:
The title should be the same as the file name and should be focused only on the topic being discussed.

```suggestion
Title: 'Input Output Redirection'
```